src/Object/Api/Twitter/User.php

```diff
@@ -127,27 +140,27 @@ public function __construct(array $publicContact, array $apcontact = [], array $
 		unset($this->withheld_scope);
 
 		// Deprecated
-		$this->profile_image_url              = $userContact['avatar'] ?? $publicContact['avatar'];
-		$this->profile_image_url_profile_size = $publicContact['thumb'];
-		$this->profile_image_url_large        = $publicContact['photo'];
+		$this->profile_image_url              = Contact::getAvatarUrlForUrl($publicContact['url'], $uid, Proxy::SIZE_MICRO);
+		$this->profile_image_url_profile_size = Contact::getAvatarUrlForUrl($publicContact['url'], $uid, Proxy::SIZE_THUMB);
+		$this->profile_image_url_large        = Contact::getAvatarUrlForUrl($publicContact['url'], $uid, Proxy::SIZE_LARGE);
```

Comment:
Same as above, you could overwrite `$publicContact['thumb']` etc. at the factory